comps/submitbtn.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+
+function SubmitBtn({submitAnswers, max, first, last, list, id}){
+
+	var text = "Volgende"
+
+	if (last === max){
+		text = "Inleveren"
+	}
```

Comment:
```suggestion
	const buttonText = last === max ? "Inleveren" : "Volgende"
```
Try to prefer `const` over `let` over `var` (actually, try never to use var)